apps/studio/components/interfaces/ProjectCreation/RegionSelector.tsx

```diff
@@ -35,9 +35,14 @@ export const RegionSelector = ({ cloudProvider, field }: RegionSelectorProps) =>
   const availableRegions = getAvailableRegions(PROVIDERS[cloudProvider].id)
   const regionsArray = Object.entries(availableRegions)
 
-  const { isLoading: isLoadingDefaultRegion } = useDefaultRegionQuery({
-    cloudProvider,
-  })
+  const { isLoading: isLoadingDefaultRegion } = useDefaultRegionQuery(
+    {
+      cloudProvider,
+    },
+    {
+      refetchOnWindowFocus: false,
+    }
```

Comment:
i don't think doing this here will have any difference though, as this isn't mutating the form values

Reply:
this is probably where it might make more sense, but the refetchOnWindowFocus is already false
https://github.com/supabase/supabase/blob/d4c928ca72dcae65cfd4b072b3c8a8aea06cd8dd/apps/studio/pages/new/%5Bslug%5D.tsx#L168-L177